src/Components/Auth/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '../../FormStyles.css';
+import { Formik, Field, Form } from 'formik';
+import { SignupSchema } from './SignupSchema';
+import { Alert } from '@mui/material';
+
+const RegisterForm = () => {
+  const initialValues = {
```

Comment:
initialValues is a too generic name, please use one more descriptive like userData or user